drivers/include/periph/flashpage.h

```diff
@@ -85,6 +85,15 @@ extern "C" {
 #define FLASHPAGE_WRITE_BLOCK_ALIGNMENT
 #endif
 
+/**
+ * @def FLASHPAGE_ERASE_STATE
+ *
+ * @brief   State of an erased byte in memory
+ */
+#ifdef DOXYGEN
+#define FLASHPAGE_ERASE_STATE
```

Comment:
How about 

```suggestion
#if defined(DOXYGEN) || !defined(FLASHPAGE_ERASE_STATE)
#define FLASHPAGE_ERASE_STATE           (0xFFU)
```

so we only have to define it if the CPU does not match the default behavior.